packages/react-positioning/src/utils/debounce.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Promise microtask debouncer used by Popper.js v2
+ * https://github.com/floating-ui/floating-ui/blob/v2.x/src/utils/debounce.js
+ * @param fn function that will be debounced
+ */
+export function debounce<T>(fn: Function): () => Promise<T> {
```

Comment:
I assume the need of this is that v3 doesn't ship it anymore ? if that's the case can we add the comment about that as well ?

Reply:
Added in [ef658f4](https://github.com/microsoft/fluentui/pull/21828/commits/ef658f4645a1043ee2e858cf706c7a63407f5b29)